src/gui/editors/AutomationEditor.cpp

```diff
@@ -646,11 +642,29 @@ void AutomationEditor::mousePressEvent( QMouseEvent* mouseEvent )
 
 						Engine::getSong()->setModified();
 					}
+					else // If we didn't click an outValue
+					{
+						// We check if the quantized position of the time we clicked has a
+						// node and set its outValue
+						MidiTime quantizedPos = Note::quantized(MidiTime(posTicks),
+										m_pattern->quantization());
```

Comment:
Let's not use multiple tabs to align. If we want to do it that way, this is my suggested way.
```suggestion
						MidiTime quantizedPos = Note::quantized(
							MidiTime(posTicks),
							m_pattern->quantization()
						);
```